src/resources/particles.ts

```diff
@@ -1,89 +1,77 @@
-import { tsParticles } from "tsparticles-engine";
-import { loadLinksPreset } from "tsparticles-preset-links";
+import { tsParticles } from "@tsparticles/engine";
+import { loadLinksPreset } from "@tsparticles/preset-links";
 import { DEFAULT_PRIMARY_COLOR } from "./styles-data";
 
 loadLinksPreset(tsParticles).then(() => {
-  tsParticles.load("particles", {
```

Comment:
How does it know what element to use now?

Reply:
I guess it would require to add `id: "particles"` as option.

Reply:
seems to be that "particles" is the default value. But I will add it